### PR TITLE
HS-302: Update README

### PR DESCRIPTION
**Pull request by indigo423**
Adding some more detailed information on what the project goal is and providing some very basic about the license, where to find help, and documentation.

## Jira link(s)
https://issues.opennms.org/browse/HS-302

## Checklist
* [x] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [x] Appropriate reviewer(s) have been selected.
* [x] Jira issue(s) have been updated to "In Review".
* [x] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [x] Documentation has been updated as necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
-# Horizon Stream
+# Horizon Stream 🚀
 
-**Horizon Stream** is a new distribution of OpenNMS, inspired from the existing platform
+Horizon Stream is the future of OpenNMS Horizon, an open source network monitoring.
```
Comment (Bonrob2) [inline README.md:3]:
```suggestion
Horizon Stream is the future of OpenNMS Horizon, an open source network monitoring platform.
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+It aims to be an easy to use, secure, reliable and extensible open-source network monitoring plattform.
+The goal is to give our users network visibility and actionable insights helping them to provide better IT services. 
 
-See [Getting Started on the Wiki](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started) for development instructions.
+☣️ The project is in a very early stage and things break and move around. It is not ready to run production workloads.
```
Comment (Bonrob2) [inline README.md:7]:
```suggestion
☣️ The project is in a very early stage and things break and move around.
It is not ready to run production workloads.
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
 
-See [PLATFORM](platform/README.md) for details on the work in progress.
+## Supported Platforms
+
+To deploy the whole application a [Kubernetes (K8s)](https://kubernetes.io/) cluster is required.
+As it is right now you need a K8s cluster with 24GB RAM.
```
Comment (Bonrob2) [inline README.md:12]:
```suggestion
Currently you need a K8s cluster with 24GB RAM.
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,36 @@
...
+
+## Documentation
+
+We use the GitHub Wiki in the Horizon Stream GitHub project for documentation of the project.
```
Comment (Bonrob2) [inline README.md:16]:
```suggestion
We use the GitHub Wiki in the Horizon Stream GitHub project to document the project.
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,36 @@
...
+It gives us enough flexibility and structure to migrate it later to a more sophisticated versioned documentation framework.
+You find the documentation how to get it installed in the [administrators getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started---Admin) section.
```
Comment (Bonrob2) [inline README.md:18]:
```suggestion
The following sections might be of most interest: 
* Installation in the [administrators getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started---Admin) section.
```

File: README.md [hunk 6]
```diff
@@ -1,7 +1,36 @@
...
+If you are interested in building it from source you find instruction in our [developers getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started) chapter.
```
Comment (Bonrob2) [inline README.md:19]:
```suggestion
* Building from source  in the [developers getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started) section.
```

File: README.md [hunk 7]
```diff
@@ -1,7 +1,36 @@
...
+In case you want to contribute code you can find instructions in our [contribution guidelines](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines).
```
Comment (Bonrob2) [inline README.md:20]:
```suggestion
* [Contribution guidelines](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines), if you want to contribute code.
```

File: README.md [hunk 8]
```diff
@@ -1,7 +1,36 @@
...
+
+## Documentation
+
+We use the GitHub Wiki in the Horizon Stream GitHub project for documentation of the project.
+It gives us enough flexibility and structure to migrate it later to a more sophisticated versioned documentation framework.
+You find the documentation how to get it installed in the [administrators getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started---Admin) section.
+If you are interested in building it from source you find instruction in our [developers getting started](https://github.com/OpenNMS/horizon-stream/wiki/Getting-Started) chapter.
+In case you want to contribute code you can find instructions in our [contribution guidelines](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines).  
+
+In case you run into problems, you can look into [troubleshooting](https://github.com/OpenNMS/horizon-stream/wiki/Troubleshooting) for quick help.
```
Comment (Bonrob2) [inline README.md:22]:
```suggestion
* [Troubleshooting](https://github.com/OpenNMS/horizon-stream/wiki/Troubleshooting) for quick help.
```

File: README.md [hunk 9]
```diff
@@ -1,7 +1,36 @@
...
+If you have questions feel free and talk to people in our community in our [opennms-discussion](https://chat.opennms.com/opennms-discuss) channel in Mattermost.
+
+## Questions
+
+### How does this relate to OpenNMS Horizon?
+
+To get these principles addressed without breaking people's current solutions and implementations.
+We know people in the OpenNMS community and commercial customers spent time and money to integrate OpenNMS Horizon into their environments. OpenNMS Horizon gets the maintenance and enhancements through the OpenNMS Plugin API extensions without a change.
```
Comment (Bonrob2) [inline README.md:30]:
```suggestion
We know people in the OpenNMS community and commercial customers spent time and money to integrate OpenNMS Horizon into their environments.
OpenNMS Horizon gets the maintenance and enhancements through the OpenNMS Plugin API extensions without a change.
```

File: README.md [hunk 10]
```diff
@@ -1,7 +1,36 @@
-# Horizon Stream
+# Horizon Stream 🚀
```
Comment (Bonrob2) [inline README.md:1]: Need to determine if stream is the name we are going to use even at this early stage. That is a marketing call.

File: README.md [hunk 11]
```diff
@@ -1,7 +1,36 @@
...
 
-**Horizon Stream** is a new distribution of OpenNMS, inspired from the existing platform
+Horizon Stream is the future of OpenNMS Horizon, an open source network monitoring.
+It aims to be an easy to use, secure, reliable and extensible open-source network monitoring plattform.
```
Comment (Bonrob2) [inline README.md:4]: These two lines require a bit of messaging. They imply that Horizon doesn't already give them this. How can we position Stream effectively?

File: README.md [hunk 12]
```diff
@@ -1,7 +1,36 @@
...
+In case you want to contribute code you can find instructions in our [contribution guidelines](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines).  
+
+In case you run into problems, you can look into [troubleshooting](https://github.com/OpenNMS/horizon-stream/wiki/Troubleshooting) for quick help.
+If you have questions feel free and talk to people in our community in our [opennms-discussion](https://chat.opennms.com/opennms-discuss) channel in Mattermost.
+
+## Questions
+
+### How does this relate to OpenNMS Horizon?
+
+To get these principles addressed without breaking people's current solutions and implementations.
```
Comment (Bonrob2) [inline README.md:29]: What principles are we addressing?

File: README.md (hunk 9 above)
Comment (Bonrob2) [reply README.md:30]: Also, what maintenance and enhancements? From Stream?

File: README.md [hunk 13]
```diff
@@ -1,7 +1,36 @@
...
+We know people in the OpenNMS community and commercial customers spent time and money to integrate OpenNMS Horizon into their environments. OpenNMS Horizon gets the maintenance and enhancements through the OpenNMS Plugin API extensions without a change.
+We can't do a major re-architecture of the platform without breaking things for our current users.
```
Comment (Bonrob2) [inline README.md:31]: It is not clear from earlier that we are doing a re-architecture of the platform. Suggest being clear about that near the beginning.

File: README.md [hunk 14]
```diff
@@ -1,7 +1,36 @@
...
+Our goal is to take over the OpenNMS Plugin API as an integration contract, and we aim to make the existing plugins compatible.
```
Comment (Bonrob2) [inline README.md:32]:
```suggestion
Our goal is to take over the OpenNMS Plugin API as an integration contract, and we aim to make the existing plugins compatible.
```
Integration with Horizon?

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
A few updates and comments on the messaging.

**Comment by j-white** [comment]
@indigo423 what's the next step here?

**Pull request closed without merging**